lib/stsci/tools/capable.py

```diff
@@ -10,6 +10,7 @@
 
 import os, sys
 PY3K = sys.version_info[0] > 2
+descrip = "basic capabilities file, last edited 29 July 2016"
```

Comment:
Agreed the title is vague - wasn't sure anyone but me was watching this repo.  I'll do better in the future.  This is a test of our test system.  Since we repeatedly have machines that are not git-or-svn updating before running the tests (due to test-infrastructure errors), this will show if that occurred.  I plan to update the date in this string on occasion.